DeltaEngine/Engine/Rendering/SpriteMap.hpp

```diff
@@ -17,8 +17,7 @@ struct SpriteMapEntry // DTO
 class SpriteMap
 	{
 	public:
-		SpriteMap(const std::string& name, const std::string& spritePath, std::vector<SpriteMapEntry> sprites)
-			: _name{ name }
+		SpriteMap(const std::string& category, const std::string& spritePath, std::vector<SpriteMapEntry> sprites)
```

Comment:
Alle plaatjes in eenzelfde map zijn nu bound aan een category, dit lijkt me geen probleem. Als dit wel een probleem is moet toch wel per sprite aan kunnen geven wat de categorie is

Reply:
dat klopt heb ik een kleine verandring voor gemaakt. waardoor dit wel gefixt is. dus nu is de naam ook de category das ook met de noormale sprites als goed is.